image-sync/requirements.txt

```diff
@@ -0,0 +1,4 @@
+git+https://github.com/enolfc/atrope@catchall
```

Comment:
Note: we should perhaps move this to https://github.com/EGI-Federation/ instead?

Reply:
yes, or contribute upstream. There needs to be more changes in atrope to have better caching and management of images if this starts to get traction